src/NuGet.Core/NuGet.CommandLine.XPlat/Commands/ConfigCommands/ConfigCommand.cs

```diff
@@ -0,0 +1,45 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using Microsoft.Extensions.CommandLineUtils;
+using NuGet.Common;
+
+namespace NuGet.CommandLine.XPlat
+{
+    internal partial class ConfigCommand
```

Comment:
We were using a template to generate the code before so that's why it had a partial keyword; I'll remove it since we're no longer using the template tool.